pennylane/qnode.py

```diff
@@ -522,11 +522,12 @@ def tape(self) -> QuantumTape:
     def construct(self, args, kwargs):
         """Call the quantum function with a tape context, ensuring the operations get queued."""
 
-        self._tape = qml.tape.QuantumTape()
-
-        with self.tape:
+        with qml.queuing.AnnotatedQueue() as q:

```

Comment:
Think we could assign `_qfunc_output` in `make_qscript`? Is that the one thing blocking us from using it here?

Reply:
that was the blocker. I'm good with that, will add it